DevelopmentWorkflow.md

```diff
@@ -0,0 +1,78 @@
+# Development Workflow
+
+<!-- TOC depthFrom:2 -->
+
+- [Running locally](#running-locally)
+    - [Prerequisites](#prerequisites)
+    - [Install dependencies](#install-dependencies)
+    - [Run the demo app](#run-the-demo-app)
+    - [Run the demo-angular app](#run-the-demo-angular-app)
+- [Run UI Tests](#run-ui-tests)
+- [Developer workflow](#developer-workflow)
+
+<!-- /TOC -->
+
+## Running locally
+
+### Prerequisites
+
+* Install your native toolchain and NativeScript as [described in the docs](https://docs.nativescript.org/plugins/plugins)
+
+* Review [NativeScript plugins documentation](https://docs.nativescript.org/plugins/plugins) for more details on plugins development
+
+
+### Install dependencies
+
+```
+$ cd nativescript-imagepicker/src
+$ npm install
+```
+
+### Run the demo app
+From the `src` folder
+```
+$ npm run demo.ios 
+$ npm run demo.android
+```
+
+### Run the demo-angular app
+```
+$ cd nativescript-imagepicker/demo-angular
+$ tns run ios 
+$ tns run android
+
+```
+
+## Run UI Tests
+
+1. Navigate to `demo/e2e`
+    ``` bash
+    cd demo/e2e
+    ```
+
+2. Make sure to have an emulator set up or connect a physical Android/iOS device.
+
+3. Build the app for Android or iOS
+    ```bash
+    tns run android/ios
```

Comment:
if only building, then command could be changed to 
```bash
tns build android
# or
ths build ios
```
Someone might try running `tns run android/ios` and get `The input is not valid sub-command for 'run' command`.